layouts/shortcodes/demo_description.md

```diff
@@ -0,0 +1,24 @@
+The [official OpenTelemetry
```

Comment:
When reading the page, this sentence feels a bit too repetitive compared to the previous sentence (the description). Maybe this can be merged with the previous one.